rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ApiDependenciesData.java

```diff
@@ -0,0 +1,205 @@
+package org.jboss.windup.rules.apps.mavenize;
+
+/**
+ * This class was originally intended as a static store of various BOM's dependencies.
+ * But it turned out there are so many that it's better to extract them right from the BOM.
+ * That is implemented in nexus-repo-indexer 6.
+ *
+ * This might be deleted in the final version.
+ *
+ * @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
+ */
+interface ApiDependenciesData
```

Comment:
Yeah... I thought it would ony be like 10 things but then I saw that each technology has around 5 artefacts... and there are dozens of techs.